features/items.md

```diff
@@ -0,0 +1,237 @@
+# Items
+
+## Introduction
+OpenHAB has a strict separation between the physical world (the “things”, see below) and the application, which is built around the notion of “items” (also called the virtual layer).
+
+Items represent functionality that is used by the application (mainly user interfaces or automation logic). Items have a state and are used through events. They are objects that can be read from, or written to, in order to interact with them.
```

Comment:
A general note: When working with git, differences are detected by line. That's why, it's sometimes annoying to grasp the change in a markdown document, if you only changed one spelling error. Example: https://github.com/openhab/openhab-docs/pull/42/files
That's why, it is good practice, to have each sentence in a single line. ([yet another example](https://raw.githubusercontent.com/pabloNZ/openhab-docs/gh-pages/features/sitemap.md))